lektor/admin/static/js/widgets/flowWidget.jsx

```diff
@@ -263,7 +263,7 @@ const FlowWidget = createReactClass({
               className='btn btn-default btn-xs'
               title={this.props.value[idx].collapsed ? i18n.trans('Expand') : i18n.trans('Collapse')}
               onClick={this.props.value[idx].collapsed
-                       ? this.expandBlock.bind(this, idx) : this.collapseBlock.bind(this, idx)}>
+                ? this.expandBlock.bind(this, idx) : this.collapseBlock.bind(this, idx)}>
```

Comment:
May I suggest breaking this up instead?
```js
const onClickHandler = this.props.value[idx].collapsed ? this.expandBlock : this.collapseBlock;
...
onClick={onClickHandler.bind(this, idx)}
```

or maybe even:

```js
const isCollapsed = this.props.value[idx].collapsed;
const onClickHandler = isCollapsed ? this.expandBlock : this.collapseBlock;
...
onClick={onClickHandler.bind(this, idx)}
```

Reply:
This change was made by `standard --fix` - I didn't really want to do any non-trivial code changes in this PR ;)

Reply:
Fair enough, I figured that was the case :)